gensim/models/word2vec.py

```diff
@@ -452,6 +453,9 @@ def __init__(
         self.total_train_time = 0
         self.sorted_vocab = sorted_vocab
         self.batch_words = batch_words
+        
+        self.load = methodize(load, self)
```

Comment:
Did you mean `self.load = methodize(self.load, self)` ?
it is very confusing to assign `self.load` and then define `def load`